pkg/domain/infra/abi/play.go

```diff
@@ -314,8 +319,18 @@ func (ic *ContainerEngine) PlayKube(ctx context.Context, body io.Reader, options
 		return nil, fmt.Errorf("YAML document does not contain any supported kube kind")
 	}
 
+	// Read the report and print it if Wait is set to true
+	// This ensures that the report information is printed when --wait is set in kube play
+	// and we are waiting for the service container to finish or an interrupt signal
+	if options.Wait {
```

Comment:
I still have issues with  printing here as it is not working for the remote client.

I didn't check the tests but it suggests the tests are insufficient and not checking for the reports to be printed.

Doesn't that require moving the waiting logic to `cmd/podman/kube`?

Reply:
> I couldn't figure out a way of having podman-remote print the report from the PlayKube function, if you have any ideas I would be happy to get that done.

The only way I see is to wait in `cmd/podman`. `PlayKube` may still wait when being run inside of systemd but that's not an issue as it's not interacting with a human.

I think we can get there fairly easily by:
* Instead of passing a `Wait` field as an option, we can return a `ServiceContainer` field in the report.
* Then in `cmd/podman` when `--wait` is set we use the `ServiceContainer` field to call `ContainerWait(report.ServiceContainer)`.

That would have the same effect but it allows for printing the reports in the "frontend" and would work for the local and remote case.

Reply:
Got it! Fixed.